src/components/Search.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { DocSearch } from '@docsearch/react';
+
+import '@docsearch/css';
+
+function Search() {
+  return (
+    <DocSearch
+      appId="UB8IN95AB5"
+      indexName="kurl-algolia-config"
+      apiKey="46812184318efc4cb48cdb423ba2d498"
```

Comment:
Is it safe to have the API key here? Is this something we could put in an environment variable?

Reply:
That's the Search API key, which Algolia says can safely be used in frontend code. Still probably smart to add them as environment variables, so I'll do that. Thanks Star!